packages/datagrid/src/types/types.d.ts

```diff
@@ -3,5 +3,4 @@ declare module '*.scss' {
 	export default content;
 }
 
-declare module '@talend/react-components';
```

Comment:
the goal is to remove this so typings are used.
Note this is a good escape in case you are not able to use react-components typings